src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -117,7 +121,7 @@ public Envelope getEnvelopeByFileAndContainer(String container, String zipFileNa
      * @param envelope to assert against
      * @param pdfs     to assert against
      */
-    public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs) {
+    public void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs) {
```

Comment:
unrelated, unnecessary change imho

Reply:
more validation coming soon, let's leave it as is, I'll move validation to a separate class on next PR once this is merged.